src/Mods.jl

```diff
@@ -159,6 +162,12 @@ julia> 92%14
 
 julia> CRT(BigInt, Mod{9223372036854775783}(9223372036854775782), Mod{9223372036854775643}(9223372036854775642))
 ```
+
+!!! note
+
+    `CRT` uses `BigInt` by default to prevent potential integer overflow.
+    If you are confident that numbers does not overflow in your application,
+    please specify an optional type parameter as the first argument.
 """
 function CRT(remainders, primes) where T
```

Comment:
There is no default `BigInt` argument, opposed to the note.

Reply:
Added the type argument in the new commit.